modules/core/include/opencv2/core/ocl.hpp

```diff
@@ -278,55 +278,52 @@ class CV_EXPORTS Platform
     Impl* p;
 };
 
-/*
-//! @brief Attaches OpenCL context to OpenCV
+/**
+// @brief Attaches OpenCL context to OpenCV
```

Comment:
There is no need in `//` at the beginning of each line of multi-line comment, please use one of the following formats:
```.cpp
/**
  .....
  .....
  .....
*/
```
or
```.cpp
//! ...
//! ...
//! ...
//! ...
```
Usually we use the first variant: [example](http://docs.opencv.org/master/d4/db1/tutorial_documentation.html#tutorial_documentation_quick_start_2).